openide.loaders/src/org/openide/loaders/TemplateWizard2.java

```diff
@@ -392,7 +394,39 @@ private void fireStateChanged () {
     private javax.swing.JTextField newObjectName;
     // End of variables declaration//GEN-END:variables
 
+    private DataFolder findRelativeFolder(File fd) {
```

Comment:
I am surprise we have to work with `File`. If we could operated only on `FileObject`, then there is [FileUtil.getRelativePath](http://bits.netbeans.org/dev/javadoc/org-openide-filesystems/org/openide/filesystems/FileUtil.html#getRelativePath-org.openide.filesystems.FileObject-org.openide.filesystems.FileObject-).

Reply:
well ... is it a hint :) what should I do ?
Internally the Wizard converts `FileObject` to `File` so it can pass the directory to Property Editor. I used Paths since `Path.relativize` seems more powerful - it operates on arbitrary paths, not just on folder and file as `FileUtil.getRelativePath` does.